src/Lykke.Service.Operations/Workflow/Sagas/BlockchainCashoutSaga.cs

```diff
@@ -138,7 +138,36 @@ public async Task Handle(SolarCashOutCompletedEvent evt, ICommandSender commandS
 
             commandSender.SendCommand(command, "operations");
         }
-        
+
+        #region CashourProcessorEvents
+
+        [UsedImplicitly]
+        public async Task Handle(CashoutsBatchCompletedEvent evt, ICommandSender commandSender)
+        {
+            _log.Info($"CashoutsBatchCompletedEvent for BIL received. BatchId [{evt.BatchId}]", evt);
```

Comment:
Looks like, exsessive logging, isn't it? Usually Cqrs-engine logs entry events/commands

Reply:
Just added what was before in operations service. 
Need one person to confirm that we should delete such log entrances

Reply:
We should delete this, if message logging is not disabled manually in CqrsEngine. Check it and remove such logging if Cqrs loggind is enabled